code/CMakeLists.txt

```diff
@@ -193,6 +193,8 @@ SET( Common_SRCS
   Common/ScenePreprocessor.cpp
   Common/ScenePreprocessor.h
   Common/SkeletonMeshBuilder.cpp
+  Common/StackAllocator.h
```

Comment:
PLease inline the implementation in an inl-file.

Reply:
done